extra/1-currency-conversion.js

```diff
@@ -15,7 +18,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(amount) {
+  let rate = 5.7;
+  return parseFloat((amount * 5.7 * 0.99).toFixed(2));
```

Comment:
Math.round(priceInBRL * 100) / 100; is used in 
JavaScript-Core-1-Coursework-Week1-Solution